.github/workflows/ci.yml

```diff
@@ -201,7 +201,7 @@ jobs:
         with:
           # If you bump the version, also update docs/contributing.md
           # and all other workflows that install uv
-          version: "0.5.1"
+          version: "0.7.3"
```

Comment:
These would also all need to be reverted.

(BTW, for Cargo.lock another option would be just to run Dependabot after adapting to the `gix` and `rand` changes. We do update uv.lock manually, though)